### PR TITLE
chore(deps): automate update of java formatter

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

A new version allows to simplify the call to the formatter, but I missed the update because renovate wasn't setup for it yet.

### CODE REVIEW

**Review by shortcuts — COMMENTED** [review]
noice!

File: renovate.json [hunk 1]
```diff
@@ -49,6 +49,18 @@
         "\\s\"(?<depName>.*?)\": \"(?<currentValue>.*?)\",?\\s"
       ],
       "datasourceTemplate": "npm"
+    },
+    {
+      "description": "Update the java formatter",
+      "fileMatch": [
```
[the remark this answers is not in the document]
Comment (shortcuts) [reply renovate.json:55]: That's what I expected too when added the support for mustache files but it was in fact matching filenames 🤔 we can try and see how it goes on Friday I guess!

**Review by shortcuts — APPROVED** [review]
Maybe revert the updates and only add the renovate part, so we can know on Friday if it works!

Otherwise good to go :D

**Comment by millotp** [comment]
good idea !

File: renovate.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (millotp) [reply renovate.json:55]: we can check right after merging by checking the box in #532 😉

**Review by shortcuts — DISMISSED** [review]

**Pull request merged**